packages/esm-service-queues-app/src/queue-table/queue-table-by-status-skeleton.component.tsx

```diff
@@ -2,31 +2,31 @@ import { DataTableSkeleton, SkeletonText, Tab, TabList, TabPanel, TabPanels, Tab
 import { isDesktop, useLayoutType } from '@openmrs/esm-framework';
 import React from 'react';
 import styles from './queue-table.scss';
+import classNames from 'classnames';
 
 export const QueueTableByStatusSkeleton = () => {
   const layout = useLayoutType();
 
   return (
     <div className={styles.container}>
-      <div className={styles.headerContainer}>
-        <div className={!isDesktop(layout) ? styles.tabletHeading : styles.desktopHeading}>
-          <h3>
-            <SkeletonText />
-          </h3>
-        </div>
+      <div className={styles.statusTableContainer}>
```

Comment:
This seems to make the assumption that there are always 3 statuses.  If you are doing what I think you are doing here, you should add a DataTableSkeleton for each status configured on the queue, not just 3 arbitrarily.